tests/test_examples_sklearnex.py

```diff
@@ -53,7 +53,15 @@ def testit(self):
     print("Generating tests for " + os.path.splitext(file)[0])
 
 
-files = [f for f in os.listdir(examples_path) if f.endswith(".py")]
+filters = [lambda x: 'spmd' not in str(x), lambda x: str(x).endswith(".py")]
+
+
+def joint_filter(x):
+    return all([f(x) for f in filters])
+
+
+files = list(filter(joint_filter, os.listdir(examples_path)))
+
```

Comment:
Can this work better?
```suggestion
files = [f for f in os.listdir(examples_path) if f.endswith(".py") and 'spmd' not in f]
```

Reply:
rejected

Reply:
This construction is much complicated than just one `and`